src/material/radio/radio.ts

```diff
@@ -516,8 +516,12 @@ export abstract class _MatRadioButtonBase extends _MatRadioButtonMixinBase imple
   }
 
   /** Focuses the radio button. */
-  focus(options?: FocusOptions): void {
-    this._focusMonitor.focusVia(this._inputElement, 'keyboard', options);
+  focus(origin?: FocusOrigin, options?: FocusOptions): void {
```

Comment:
I believe that this would be a breaking change. The `origin` would have to be added as the second parameter for it to be non-breaking.

Reply:
Oh yes, let me make origin the second param. Thanks!